composer.json

```diff
@@ -30,13 +30,49 @@
         }
     },
     "extra": {
-        "class": "ComposerGpgVerify\\Verifier",
+        "class": "ComposerGpgVerify\\Verify",
         "branch-alias": {
             "dev-master": "2.0.x-dev"
+        },
+        "composer-gpg-verify": {
+            "allow-unsigned": [
+                "composer/ca-bundle",
+                "composer/composer",
+                "composer/semver",
+                "composer/spdx-licenses",
+                "psr/log",
+                "justinrainbow/json-schema",
+                "seld/cli-prompt",
+                "seld/jsonlint",
+                "seld/phar-utils",
+                "symfony/console",
+                "symfony/debug",
+                "symfony/filesystem",
+                "symfony/finder",
+                "symfony/polyfill-mbstring",
+                "symfony/process"
+            ],
+            "allow-untrusted": [
```

Comment:
Yeah, this section is a TODO. I think I will just add an exclusion list, a warning, and eventually a list of keybase usernames to trust.